app/containers/TwoFactor/index.tsx

```diff
@@ -123,6 +142,12 @@ const TwoFactor = React.memo(() => {
 		setData({});
 	};
 
+	useEffect(() => {
+		if (data.invalid) {
+			setError('code', { message: I18n.t('Invalid_code'), type: 'validate' });
+			AccessibilityInfo.announceForAccessibility(I18n.t('Invalid_code'));
+		}
+	}, [data.invalid]);
```

Comment:
Is it really necessary to put data on a useEffect's dependency, so it can trigger an error?
Isn't it possible to do it right next to where data is actually set?